modules/backend/widgets/Filter.php

```diff
@@ -566,7 +566,11 @@ protected function makeFilterScope($name, $config)
         /*
          * Set scope value
          */
-        $scope->value = $this->getScopeValue($scope);
+        if($scopeType=='checkbox' || $scopeType=='switch'){
```

Comment:
Please use strict type comparisons (`===`), not loose type comparisons.

Reply:
@LukeTowers  done